resolwe_bio/management/commands/generate_samples.py

```diff
@@ -0,0 +1,248 @@
+                                  'molecule': 'polyA RNA',
+                                  'description': 'Expression profiling'},
+                          'protocols': {
+                            'fragmentation_method': 'sonication',
+                            'growth_protocol': 'Cells were grown in nutrient media (HL-5) to mid-log phase prior to collection for development.',
+                            'treatment_protocol': 'Development',
+                            'library_prep': "TriZol (Life Sciences) -- Cells were scraped (filter) or pelleted (suspension) and disrupted in TriZol. Total RNA was extracted by phenol/chloroform as per manufacturer's instructions. Ribosomal RNA was depleted using the InDA-C method (Ovation, Nugen) reagents, following manufacturer's instructions."}}
+        if name.startswith('Hs'):
+            annotation = {'geo': {'organism': 'Homo sapiens',
+                                  'annotator': random.choice(annotator),
+                                  'source': 'brain',
+                                  'cell_type': random.choice(['neuron', 'glia']),
+                                  'genotype': random.choice(['wildtype', 'CDK6-/-']),
+                                  'experiment_type': 'RNA-Seq',
+                                  'molecule': 'polyA RNA',
+                                  'optional_char': ['Cell line: DBTRG-05MG'],
+                                  'description': "Human brain study. Expression profiling."},
+                          'protocols': {
+                            'fragmentation_method': 'sonication',
+                            'growth_protocol': growth_protocol,
+                            'library_prep': library_prep}}
+        if name.startswith('Mm'):
+            annotation = {'geo': {'organism': 'Mus musculus',
+                                  'annotator': random.choice(annotator),
+                                  'source': 'brain',
+                                  'genotype': random.choice(['wildtype', 'CDK6-/-']),
+                                  'cell_type': random.choice(['neuron', 'microglia']),
+                                  'experiment_type': 'RNA-Seq',
+                                  'molecule': 'polyA RNA',
+                                  'optional_char': ['Cell line: EOC 2'],
+                                  'description': "Mouse brain study. Expression profiling."},
+                          'protocols': {
+                            'fragmentation_method': 'sonication',
+                            'growth_protocol': growth_protocol,
+                            'library_prep': library_prep}}
+        return annotation
+
+    def create_data(self, reads_name='seq_reads', annotated=False):
+        # get test data paths
+        data_path = settings.FLOW_EXECUTOR['DATA_PATH']
+        path = os.path.abspath(os.path.join(os.path.dirname(__file__), '..', '..'))
```

Comment:
You can probably add `tests` and `files` to this path and name it `tests_files_path`.